learning/features/assetlib/using_assetlib.rst

```diff
@@ -0,0 +1,178 @@
+issues. They are not recommended for use in serious projects, but you are
+encouraged to download, test them, and submit issues to the original authors.
+
+You can mix and match any of the search filters and criteria, and upon clicking
+the Search button, receive the list of all assets in the Library that match them.
+
+|image1|
+
+Note that the search results are not updated in real-time, so you will have to
+re-submit the search query each time you change the query settings.
+
+Breakdown of an asset
+~~~~~~~~~~~~~~~~~~~~~
+
+Now let's take a look at what an asset's page looks like and what it contains.
+
+|image2|
+
+1. Asset's thumbnail/icon.
+2. Asset's name.
+3. Current version number of the asset.
+4. Asset's average rating, displayed in stars. (This is currently unimplemented.)
+5. Asset's category, Godot version, and support status.
+6. Asset's original author/submitter.
+7. The license the asset is distributed under.
+8. The date of the asset's latest edit/update.
+9. A textual description of the asset.
+10. Links related to the asset (download link, file list, issue tracker).
+11. A SHA-256 of the asset, for download validation purposes.
+12. Images and videos showcasing the asset.
+
+Registering and logging in
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to upload assets to the AssetLib, you need to be logged in, and to do 
+that, you need a registered user account. In the future, this may also give you
+access to other features, such as commenting on or rating the existing assets.
+You do *not* need to be logged in to browse and download the assets.
+
+The login/registration page can be accessed from the AssetLib header.
+
+|image3|
+
+From here, you can register your account, which requires a valid email address,
+a username, and a (preferably strong) password.
+
+|image4|
+
+Then, you can use your username and password to log in.
+
+|image5|
+
+This will change the look of the AssetLib header. Now you can see a feed of the
+assets (TODO: what does that mean?), an overview of your submitted assets, and
+the ability to submit new assets to the Library. 
+
+|image6|
+
+You can learn how to submit assets to the Library, and what the asset submission
+guidelines are, in the next part of this tutorial, ::ref:`doc_uploading_to_assetlib`.
+
+In the editor
+-------------
+
+You can also access the AssetLib directly from Godot. It is available from the
+project's workspaces tab (in a similar manner to the 2D, 3D and Script views):
+
+|image7|
+
+Click on it, and Godot will fetch info about the assets from the AssetLib. Once
+it's finished, you will see a window similar to what the AssetLib website looks
+like, with some differences:
+
+|image8|
+
+Similarly to the web version of the AssetLib, here you can search
+for assets by category, name, and sort them by factors such as name or edit date.
+
+Notably, you can only fetch assets for the current version of Godot you are running,
+and you cannot download demos or projects this way.
```

Comment:
As mentioned above, possible via the project manager (it makes little sense to download the project into your current one).